lib/chunkers/text_splitter.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Chunkers
+  class TextSplitter < Base
+    def initialize(delimeter: "\n\n", delimeter_function: nil, **kwargs)
+      super(**kwargs)
+
+      @delimeter = delimeter
+      @delimeter_function = delimeter_function
+    end
+
+    def split_text(text)
+      basic_split = text.split(delimeter)
```

Comment:
... and then when `MARKDOWN_DELIMITERS` are passed in -- I guess we're supposed to split multiple times for each of the delimeters? Was that the intention?

Reply:
Yeah. It wasn't working correctly with multiple, fixed now.